uint16 DOF
string[] names
Pose[] poses